src/containers/Tenant/Diagnostics/Overview/TopicStats/TopicStats.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../../../../styles/mixins.scss';
+
+.ydb-overview-topic-stats {
+    &__title {
+        @include info-viewer-title();
+    }
+
+    .ydb-loader {
+        margin-top: 50px;
+    }
+
+    .ydb-bars {
+        margin-top: -5px;
+    }
+
+    &__info {
+        .info-viewer__label-text {
+            display: flex;
+            overflow: visible;
+
+            max-width: 150px;
+
+            white-space: normal;
+        }
```

Comment:
in my previous story about defining styles for an element and not for a nested block I didn't realize this was styles for an element of info-viewer, not for the root of info-viewer.

but now I have another question: are these styles to make labels multiline?
seems like it should be an InfoViewer internal behaviour enabled by a prop, not an outside modification